general/cisetup/cisetup.go

```diff
@@ -457,58 +450,67 @@ func (cc *CiSetupCommand) getPiplinesCompletionInstruction(pipelinesFileName str
-			" Inside the 'environment' section, set the value of the M2_HOME variable,",
-			" to the Maven installation directory on the Jenkins agent (the directory which includes the 'bin' directory).")
+			"* Inside the 'environment' section, set the value of the M2_HOME variable,",
+			"  to the Maven installation directory on the Jenkins agent (the directory which includes the 'bin' directory).")
 	}
 
 	JenkinsCompletionInstruction = append(JenkinsCompletionInstruction,
-		" Inside the 'environment' section, set the value of the JFROG_CLI_BUILD_URL variable,",
-		" so that it includes the URL to the job run console log in Jenkins. You may need to look at URLs of other job runs, to build the URL.",
+		"* Inside the 'environment' section, set the value of the JFROG_CLI_BUILD_URL variable,",
+		"  so that it includes the URL to the job run console log in Jenkins.",
+		"  You may need to look at URLs of other job runs, to build the URL.",
+		"* If cloning the code from git requires credentials, modify the 'git' step as described",
+		"  in the comment inside the 'Clone' step.",
+		"* Define an environment variable named RT_USERNAME with your JFrog Platform username as its value.",
+		"* Create credentials with 'rt-password' as its ID, with your JFrog Platform password as",
+		"  its value. Read more about this here - https://www.jenkins.io/doc/book/using/using-credentials/",
+		"* Add the new Jenkinsfile to your git repository by running the following commands:",
 		"",
-		" If cloning the code from git requires credentials, modify the 'git' step as described in the comment inside the 'Clone' step.",
-		" Define the RT_USERNAME environment variable and the RT_PASSWORD as a secret environment variable in Jenkins.",
-		" These environment variables should store the credentials to the JFrog Platform.",
-		"",
-		" Add the new Jenkinsfile to your git repository by running the following commands:",
 		"\t cd "+cc.data.LocalDirPath,
 		"\t git commit -m \"Add Jenkinsfile\"",
 		"\t git push",
-		" Create a Pipelines job in Jenkins, and configure it to pull the new Jenkinsfile from git.",
-		" Run the new Jenkins job. ", "")
+		"",
+		"* Create a Pipelines job in Jenkins, and configure it to pull the new Jenkinsfile from git.",
+		"* Run the new Jenkins job. ", "")
 
-	return JenkinsCompletionInstruction, nil
+	return JenkinsCompletionInstruction
 }
 
-func (cc *CiSetupCommand) getGithubActionsCompletionInstruction(githubActionFileName string) ([]string, error) {
+func (cc *CiSetupCommand) getGithubActionsCompletionInstruction(githubActionFileName string) []string {
 	return []string{"", colorTitle("Completing the setup"),
-		"We configured the JFrog Platform and generated a " + githubActionFileName + " for you.",
+		"We configured the JFrog Platform and generated a GitHub Actions workflow file",
+		"under " + filepath.Join(cc.data.LocalDirPath, ".github", "workflows", githubActionFileName) + " for you.",
```

Comment:
I think it will be better to get file path (includes the name) from the GitHub file generator.